sb3_contrib/common/utils.py

```diff
@@ -67,3 +68,91 @@ def quantile_huber_loss(
     else:
         loss = loss.mean()
     return loss
+
+
+# TODO: write regression tests
+def conjugate_gradient_solver(
+    matrix_vector_dot_func: Callable[[th.Tensor], th.Tensor],
+    b,
+    max_iter=10,
+    residual_tol=1e-10,
+) -> th.Tensor:
+    """
+    Finds an approximate solution to a set of linear equations Ax = b
+
+    Source: https://github.com/ajlangley/trpo-pytorch/blob/master/conjugate_gradient.py
```

Comment:
you can also probably link to https://epubs.siam.org/doi/abs/10.1137/1.9781611971446.ch6
which is the one used by John Schulman in his reference implementation: https://github.com/joschu/modular_rl/blob/master/modular_rl/trpo.py#L122

"Demmel p312"